integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -40,7 +40,7 @@
 import tachyon.heartbeat.HeartbeatScheduler;
 import tachyon.job.CommandLineJob;
 import tachyon.job.JobConf;
-import tachyon.master.lineage.meta.LineageFileState;
+import tachyon.proto.JournalEntryProtos.LineageFileState;
```

Comment:
I've added back `master.lineage.meta.LineageFileState`, and added methods for converting between the two.